<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="frame_preview3_layout">
      <item>
       <widget class="QLabel" name="frame_label_3">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="frame_preview_3">
        <property name="maximumSize">
         <size>
          <width>94</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="frame_preview2_layout">
      <item>
       <widget class="QLabel" name="frame_label_2">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="frame_preview_2">
        <property name="maximumSize">
         <size>
          <width>94</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QVBoxLayout" name="animation_preview_layout">
      <item>
       <widget class="QGraphicsView" name="animation_preview">
        <property name="minimumSize">
         <size>
          <width>176</width>
          <height>95</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>176</width>
          <height>170</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="fps_selector">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="sliderPosition">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="fps_label">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>30 FPS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QToolButton" name="animation_zoom_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/Images/fit_size_32.png</normaloff>:/Icons/Images/fit_size_32.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QScrollBar" name="frame_preview_scroll">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="edit_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>443</width>
          <height>0</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Frame:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="frame_spin_box">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pen_tool_button">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw a single color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Icons/Images/pencil_32.png</normaloff>:/Icons/Images/pencil_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="rectangle_tool_button">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw a rectangle of a single color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Icons/Images/rectangle_32.png</normaloff>:/Icons/Images/rectangle_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="eraser_tool_button">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Erase the color to the background color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Icons/Images/eraser_32.png</normaloff>:/Icons/Images/eraser_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="dropper_tool_button">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get the color of a certain pixel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Icons/Images/color_dropper_32.png</normaloff>:/Icons/Images/color_dropper_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fill_tool_button">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fills a certain area with a color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Icons/Images/paint_bucket_32.png</normaloff>:/Icons/Images/paint_bucket_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">button_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="color_picker_button">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Icons/Images/color_wheel_32.png</normaloff>:/Icons/Images/color_wheel_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zoom_slider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Zoom</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="frame_preview1_layout">
      <item>
       <widget class="QLabel" name="frame_label_1">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="frame_preview_1">
        <property name="maximumSize">
         <size>
          <width>94</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="Line" name="vertical_line">
      <property name="minimumSize">
       <size>
        <width>3</width>
        <height>228</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="Line" name="horizontal_line">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>3</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>vertical_line</zorder>
   <zorder>frame_preview_scroll</zorder>
   <zorder>horizontal_line</zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_sprite"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_export_as_GIF"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="action_resize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="action_add_frame"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_new_sprite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/new_32.png</normaloff>:/Icons/Images/new_32.png</iconset>
   </property>
   <property name="text">
    <string>New Sprite</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/load_32.png</normaloff>:/Icons/Images/load_32.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/save_32.png</normaloff>:/Icons/Images/save_32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_export_as_GIF">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/gif_export_32.png</normaloff>:/Icons/Images/gif_export_32.png</iconset>
   </property>
   <property name="text">
    <string>Export as GIF</string>
   </property>
  </action>
  <action name="action_add_frame">
   <property name="text">
    <string>Add Frame</string>
   </property>
   <property name="toolTip">
    <string>Add a frame to the sprite</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/undo_32.png</normaloff>:/Icons/Images/undo_32.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/redo_32.png</normaloff>:/Icons/Images/redo_32.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_resize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Images/resize_32.png</normaloff>:/Icons/Images/resize_32.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="button_group"/>
 </buttongroups>
</ui>
